### ISSUE TITLE
what's the possible reason for very small training loss and disappointing inference result? 

### ISSUE THREAD

**Issue by Jiakui** [google/seq2seq#249]
I tried seq2seq for nmt with same generated toy data. However, while the training loss goes to 0.0001, the ouput in inference stage looks not very right. I am wondering whether it is related to over-fitting, however, I am using the same dataset for inference as training, therefore it should not be an over-fitting problem. 

Any suggestions? Should I adjust the dropout coefficients to get possible better result?

Thanks!
